site/src/docs/components/breadcrumb/accessibility.mdx

```diff
@@ -0,0 +1,17 @@
+---
+slug: '/components/breadcrumb/accessibility'
+title: 'Link - Accessibility'
+---
+
+import TabsLayout from './tabs.mdx';
+
+export default ({ children, pageContext }) => <TabsLayout pageContext={pageContext}>{children}</TabsLayout>;
+
+## Accessibility
+
+The HDS breadcrumb component is a native HTML list with native HTML links.
+
+### Pay attention to
+
+- The last element should have the attribute `aria-current="true"` if it is the current page. Automatically added in React.
+- Nav-element must have an `aria-label`. Automatically created in React.
```

Comment:
If the default value is removed, remember to remove the last sentence from here.